src/occurrences/Map.js

```diff
@@ -126,18 +139,60 @@ export const MapGeolocated = geolocated({
       }
     }
 
+    handlerInputSearch(event) {
+      this.setState({ searchValue: event.target.value });
+    }
+
+    getDataAPI() {
+      const { searchValue } = this.state;
+      fetch(
+        `https://api.mapbox.com/geocoding/v5/mapbox.places/${searchValue}.json?access_token=pk.eyJ1IjoicXVlcGxhbnRhIiwiYSI6ImNrOWRqemM2MzAycHQzaGxsd2RjMm02eGEifQ.hWRMUFwqlSf8nbEiJIE5BQ&fuzzyMatch=false&language=pt-BR`
+      )
+        .then((data) => data.json())
+        .then((json) => {
+          const location = json.features[0];
+          this.ref.current.leafletElement.fitBounds([
+            [location.bbox[1], location.bbox[0]],
+            [location.bbox[3], location.bbox[2]],
+          ]);
+        });
+    }
+
     toGoMyLocation(e) {
       e.preventDefault();
       e.stopPropagation();
-      this.ref.current.props.getLocation();
+      this.setState({ zoom: 14 });
     }
 
     render() {
       const { children, ...mapProps } = this.props;
-      const { position } = this.state;
+      const { position, searchValue, zoom } = this.state;
 
       return (
-        <Map ref={this.ref} center={position} {...mapProps}>
+        <Map newZoom={zoom} mapRef={this.ref} center={position} {...mapProps}>
```

Comment:
```suggestion
        <Map mapRef={this.ref} center={position} {...mapProps}>
```
não precisa mais do newZoom, rola de remover ele do state tb